{
  "dataType": "CVE_RECORD",
  "containers": {
    "adp": [
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2023-42962",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "partial"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-03-28T19:16:01.101640Z"
              }
            }
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-07-05T15:20:38.542Z"
        }
      }
    ],
    "cna": {
      "affected": [
        {
          "vendor": "Apple",
          "product": "iOS and iPadOS",
          "versions": [
            {
              "status": "affected",
              "version": "unspecified",
              "lessThan": "17.2",
              "versionType": "custom"
            }
          ]
        },
        {
          "vendor": "Apple",
          "product": "iOS and iPadOS",
          "versions": [
            {
              "status": "affected",
              "version": "unspecified",
              "lessThan": "16.7",
              "versionType": "custom"
            }
          ]
        }
      ],
      "references": [
        {
          "url": "https://support.apple.com/en-us/HT214035"
        },
        {
          "url": "https://support.apple.com/en-us/HT214034"
        }
      ],
      "descriptions": [
        {
          "lang": "en",
          "value": "This issue was addressed with improved checks This issue is fixed in iOS 17.2 and iPadOS 17.2, iOS 16.7.3 and iPadOS 16.7.3. A remote attacker may be able to cause a denial-of-service."
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "description": "A remote attacker may be able to cause a denial-of-service"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "286789f9-fbc2-4510-9f9a-43facdede74c",
        "shortName": "apple",
        "dateUpdated": "2024-03-28T15:39:10.717Z"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2023-42962",
    "state": "PUBLISHED",
    "dateUpdated": "2024-07-05T17:21:20.966Z",
    "dateReserved": "2023-09-14T19:05:11.484Z",
    "assignerOrgId": "286789f9-fbc2-4510-9f9a-43facdede74c",
    "datePublished": "2024-03-28T15:39:10.717Z",
    "assignerShortName": "apple"
  },
  "dataVersion": "5.1"
}